ajax/libs/videojs-markers/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "videojs-markers",
+  "filename": "videojs-markers.min.js",
+  "version": "0.7.0",
+  "description": "video marker breaks plugin for video.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/spchuang/videojs-markers.git"
+  },
+  "keywords": [
+    "videojs",
+    "tags",
+    "marker",
+    "video",
+    "player"
+  ],
+  "homepage": "http://www.sampingchuang.com/videojs-markers",
+  "author": "spchuang",
```

Comment:
I think we can add more information about the author.
e.g.
```js
  "author": {
    "name": "spchuang",
    "url": "https://github.com/spchuang"
  }
```
or `"author": "spchuang (https://github.com/spchuang)"`